modules/serverless-installing-cli-linux-ibm-z-rpm.adoc

```diff
@@ -0,0 +1,43 @@
+[id="installing-cli-linux-ibm-z-rpm-{context}"]
+== RPM Installation
+
+For Red Hat Enterprise Linux (RHEL), you can install the `kn` CLI as an RPM if you have an active {product-title} subscription on your Red Hat account.
+
+.Procedure
+
+. Enter the command:
```

Comment:
Again, please make command descriptions less generic than "enter the command", doesn't inform users what the commands do

Reply:
@ktania46 do you plan to follow up on this?

Reply:
Yes @abrennan89  I'm on it.